### PR TITLE
Add kb for editor inside scrollview

### PR DESCRIPTION
**Pull request by didiyordanova**

### CODE REVIEW

File: knowledge-base/dotnet-maui-editor-scroll-inside-scrollview.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+## Environment
+
+| Product |
+| --- |
+| Editor for .NET MAUI |
+
+## Description
+
+I want the <a href="https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/editor?view=net-maui-8.0" target="_blank">`Editor` for .NET MAUI</a> to scroll on Android when added inside a `ScrollView`.
```
Comment (yordan-mitev) [inline knowledge-base/dotnet-maui-editor-scroll-inside-scrollview.md:19]:
```suggestion
I want the <a href="https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/editor?view=net-maui-8.0" target="_blank">Editor for .NET MAUI</a> to scroll on Android when added inside a `ScrollView`.
```

File: knowledge-base/dotnet-maui-editor-scroll-inside-scrollview.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+This KB article also answers the following questions:
+- How to use the .NET MAUI `Editor`, so the editor to scroll when having more input elements on the page?
```
Comment (yordan-mitev) [inline knowledge-base/dotnet-maui-editor-scroll-inside-scrollview.md:22]: * Don't use the inline code style for the component names.
* Use a capital letter when mentioning the component's name (Editor).
*  Both comments apply to all occurrences of `Editor` and "editor" in this article.
```suggestion
How to use the .NET MAUI Editor, so the Editor scrolls when having more input elements on the page?
```

File: knowledge-base/dotnet-maui-editor-scroll-inside-scrollview.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+- How to use the .NET MAUI `Editor`, so the editor to resize to accommodate new input?
```
Comment (yordan-mitev) [inline knowledge-base/dotnet-maui-editor-scroll-inside-scrollview.md:23]:
```suggestion
- How to use the .NET MAUI Editor, so the Editor resizes to accommodate new input?
```

File: knowledge-base/dotnet-maui-editor-scroll-inside-scrollview.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+- How to make the scenarios described above to work correctly when wrapping the .NET MAUI `Editor` inside a Border?
```
Comment (yordan-mitev) [inline knowledge-base/dotnet-maui-editor-scroll-inside-scrollview.md:24]:
```suggestion
- How to make the scenarios described above work correctly when wrapping the .NET MAUI Editor inside a border?
```

File: knowledge-base/dotnet-maui-editor-scroll-inside-scrollview.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+
+## Solution
+
+To scroll the .NET MAUI `Editor` on Android when the editor is inside `ScrollView` you need to use the editor handler to access the native Android control. Then call the  `RequestDisallowInterceptTouchEvent` with parameter `false` to the parent view the editor is placed to.
```
Comment (yordan-mitev) [inline knowledge-base/dotnet-maui-editor-scroll-inside-scrollview.md:28]:
```suggestion
To scroll the .NET MAUI Editor on Android when the Editor is inside a ScrollView:

1. Use the Editor handler to access the native Android control.
1. Call the  `RequestDisallowInterceptTouchEvent` with parameter `false` to the parent view where the Editor is placed.
1. Use the Telerik .NET MAUI [`RadBorder`]({%slug border-overview%}) with a nested Editor to enable the Editor to resize when the content changes.
1. Use the Telerik .NET MAUI [`RadBorder`]({%slug border-overview%}) with a nested Editor to enable the Editor to scroll when the content grows.
```

File: knowledge-base/dotnet-maui-editor-scroll-inside-scrollview.md [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+This KB article also answers the following questions:
+- How to use the .NET MAUI `Editor`, so the editor to scroll when having more input elements on the page?
+- How to use the .NET MAUI `Editor`, so the editor to resize to accommodate new input?
+- How to make the scenarios described above to work correctly when wrapping the .NET MAUI `Editor` inside a Border?
+
+## Solution
+
+To scroll the .NET MAUI `Editor` on Android when the editor is inside `ScrollView` you need to use the editor handler to access the native Android control. Then call the  `RequestDisallowInterceptTouchEvent` with parameter `false` to the parent view the editor is placed to. 
+
+Use the Telerik .NET MAUI [RadBorder]({%slug border-overview%}) to achieve the following:
```
Comment (yordan-mitev) [inline knowledge-base/dotnet-maui-editor-scroll-inside-scrollview.md:30]:
```suggestion
```

File: knowledge-base/dotnet-maui-editor-scroll-inside-scrollview.md [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+
+* `Editor` inside `RadBorder` that resizes when content changes.
```
Comment (yordan-mitev) [inline knowledge-base/dotnet-maui-editor-scroll-inside-scrollview.md:32]: Why is this a separate list and doesn't expand on the steps described above?

Comment (yordan-mitev) [inline knowledge-base/dotnet-maui-editor-scroll-inside-scrollview.md:32]:
```suggestion
```

File: knowledge-base/dotnet-maui-editor-scroll-inside-scrollview.md [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+* `Editor` inside `RadBorder` that scrolls when content grows.
```
Comment (yordan-mitev) [inline knowledge-base/dotnet-maui-editor-scroll-inside-scrollview.md:33]:
```suggestion
```

File: knowledge-base/dotnet-maui-editor-scroll-inside-scrollview.md [hunk 9]
```diff
@@ -0,0 +1,118 @@
...
+
+### Example
```
Comment (yordan-mitev) [inline knowledge-base/dotnet-maui-editor-scroll-inside-scrollview.md:35]:
```suggestion
```

File: knowledge-base/dotnet-maui-editor-scroll-inside-scrollview.md [hunk 10]
```diff
@@ -0,0 +1,118 @@
...
+
+Here is an example that solves the questions described above:
```
Comment (yordan-mitev) [inline knowledge-base/dotnet-maui-editor-scroll-inside-scrollview.md:37]:
```suggestion
Here is an example that implements the suggested solution:
```

File: knowledge-base/dotnet-maui-editor-scroll-inside-scrollview.md [hunk 11]
```diff
@@ -0,0 +1,118 @@
...
+
+            return false;
+        }
+    }
+#endif
+}
+
+```
+
+**3.** To resize the editor to accommodate new inputs set its `AutoSize` property to `TextChanges`.
```
Comment (yordan-mitev) [inline knowledge-base/dotnet-maui-editor-scroll-inside-scrollview.md:80]:
```suggestion
**3.** To resize the Editor to accommodate new input, set its `AutoSize` property to `TextChanges`. 
```

File: knowledge-base/dotnet-maui-editor-scroll-inside-scrollview.md (hunk 7 above)
Comment (didiyordanova) [reply knowledge-base/dotnet-maui-editor-scroll-inside-scrollview.md:32]: so the first case is for android, and the second case when using the border is on iOS. We described mainly how to fix issues related in the maui editor using our border control.

**Review by yordan-mitev — APPROVED** [review]

**Pull request merged**